### PR TITLE
[CA-1143] Add bouncy castle to version 1.68

### PR DESCRIPTION
**Pull request by s-rubenstein**
**PR checklist**
- [ ] For each library you've modified here, decide whether it requires a major, minor, or no version bump. (Click [here](/broadinstitute/workbench-libs/blob/develop/CONTRIBUTING.md) for guidance)

If you're doing a **major** or **minor** version bump to any libraries:
- [ ] Bump the version in `project/Settings.scala` `createVersion()`
- [ ] Update `CHANGELOG.md` for those libraries
- [ ] I promise I used `@deprecated` instead of deleting code where possible

In all cases:
- [ ] Replace the appropriate version hashes in `README.md` and the `CHANGELOG.md` for any libs you changed with `TRAVIS-REPLACE-ME` to auto-update the version string
- [ ] Get two thumbsworth of PR review
- [ ] Verify all tests go green (CI _and_ coverage tests)
- [ ] Squash commits and **merge to develop**
- [ ] Delete branch after merge

### CODE REVIEW

File: project/Dependencies.scala [hunk 1]
```diff
@@ -150,6 +154,9 @@ object Dependencies {
   ).map(excludeGuavaJDK5)
 
   val google2Dependencies = commonDependencies ++ Seq(
+    bouncyCastle,
```
Comment (s-rubenstein) [inline project/Dependencies.scala:158]: I did the dependency mapping on intellij, which is how I got that this was the place that has bouncycastle. Will check afterwards on upgrading Rawls and Sam

**Review by jacmrob — APPROVED** [review]

**Comment by jacmrob** [comment]
What's our policy on codecov? 0.15% reduction seems fairly small and I'm also unsure what could be done differently with this PR.

**Comment by s-rubenstein** [comment]
@jacmrob I don't know if we have a policy. I will check with Callisto though, I believe they manage this repo.

File: project/Dependencies.scala (hunk 1 above)
Comment (Qi77Qi) [reply project/Dependencies.scala:158]: why do we need to add bouncyCatstle here?

Comment (s-rubenstein) [reply project/Dependencies.scala:158]: So, bouncyCastle 1.66 is a transitive dependency of one of our libraries in here. From AppSec (https://broadworkbench.atlassian.net/browse/CA-1143) bouncyCastle 1.66 has a security vulnerability. By explicitly including it, we can short-circuit the transitive dependency and make it fully 1.68, which was their suggested upgrade path.

**Review by marctalbott — COMMENTED** [review]
If we are upgrading the dependencies of the `google2` library, I'm guessing we'll need to upgrade its version hash in the changelog and readme (see [CONTRIBUTING.md](https://github.com/broadinstitute/workbench-libs/blob/develop/CONTRIBUTING.md#versioning)). 

You can also test that this properly updates bouncycastle in Rawls and Sam by publishing this locally, including the snapshot hash in Rawls and Sam, and then checking the bouncycastle version there (check out this section of [GETTING_STARTED.md](https://github.com/broadinstitute/workbench-libs/blob/develop/GETTING_STARTED.md#bootstrapping)).

File: project/Dependencies.scala (hunk 1 above)
Comment (Qi77Qi) [reply project/Dependencies.scala:158]: gotcha...can you make the version number into a variable similar to [this](https://github.com/broadinstitute/workbench-libs/blob/4df335214c35d2eb75458cf760b8807047b10f6f/project/Dependencies.scala#L11) assuming version for all these libs have to match

**Comment by Qi77Qi** [comment]
as @marctalbott mentioned, you'll need to do the `TRAVIS-REPLACE-ME` thing mentioned in contributing guide similar to [this PR](https://github.com/broadinstitute/workbench-libs/pull/630/files) in order for CI to publish a new version of the lib...And publishing it locally first, and test it out in Sam or Rawls is definitely even better

**Review by marctalbott — APPROVED** [review]

**Pull request merged**